### ISSUE TITLE
`--include-resources` doesn't include daemonsets

### ISSUE THREAD

**Issue by euven** [caas-team/py-kube-downscaler#64]
## Issue
I'm using the `--include-resources` flag, trying to include `daemonsets`, but i'm seeing the following error:
```
__main__.py: error: argument --include-resources: --include-resources argument should contain a subset of [cronjobs, deployments, horizontalpodautoscalers, rollouts, scaledobjects, stacks, statefulsets]
```

### Problem to solve

The documentation seems to mention `daemonsets` as an option to the `--include-resources` flag.

Can this be clarified?

**Comment by samuel-esp** [comment]
Hi @euven i will check this issue asap!

**Comment by euven** [comment]
Hey @sameul-esp - thanks for helping here.

I'm using the helm chart with default values for the image, tag, etc. Looks like `ghcr.io/caas-team/py-kube-downscaler:24.5.2` is in use there.

Do you need to release a new helm chart version? Or should I just use a newer image version in my values file?

**Comment by Fovty** [comment]
@euven Yes, something went wrong, thanks for pointing it out. I will create a new release as soon as the update for [new-pykube](https://github.com/caas-team/new-pykube/pull/24) is done

**Comment by euven** [comment]
Thanks @Fovty! I'll leave this one open for the time being then ;)

**Comment by Fovty** [comment]
Should now be fixed with release v24.7.0 / chart version 0.2.3
